docs/topics/images.rst

```diff
@@ -349,20 +361,21 @@ done from the Django shell:
 Changing per-tag
 ^^^^^^^^^^^^^^^^
 
-It's also possible to have different JPEG qualities on individual tags by using
-the ``jpegquality`` filter. This will always override the default setting:
+It's also possible to have different JPEG and WebP qualities on individual tags
+by using ``jpegquality`` and ``webpquality`` filters. This will always override
+the default setting:
 
 .. code-block:: html+Django
 
-    {% image page.photo width-400 jpegquality-40 %}
+    {% image page.photo width-400 quality-40 %}
```

Comment:
We need to change this example back to `jpegquality` and add an example with `webpquality`